src/mail/mail-eval.service.ts

```diff
@@ -161,8 +154,7 @@ export class MailEvalService {
         const newItem: any = {};
         const certEventRecord: QaCertEvent | QaCertEventGlobal | null =
           certRecord.qaCertEventIdentifier &&
-          (await this.returnManager().findOneBy(
-            !isSubmission ? QaCertEvent : QaCertEventGlobal,
+          (await this.returnManager().findOneBy(QaCertEvent,
```

Comment:
Context:  Currently, there is no functionality to include attachments in feedback emails. Current functionality simply sends an email with minimal content and with no attachment. The new functionality is to change the email content AND the attachments. 

Prior to this change, the sendMassEvalEmail method in MailEvalService class handled sending out emails both for evaluations and submissions.  However, this functionality (of sending submission feedback emails) was very minimal.  Given the two functionalities are different, I took out the logic of sending submission feedback emails to SubmissionProcessService. sendFeedbackReportEmail.  That change necessitated the modification of MailEvalService.sendMassEvalEmail to remove submission related logic (and retain only evaluation related logic).

The above code, makes a decision on what schema to use based on whether the logic is processing submission or evaluation emails.  Since, this method is no longer processing submission feedback emails, I simplified the logic to always use the schema for evaluations (which is the non-global schema "camdecmpswks").

Reply:
Thanks Yonatan!